src/gamma_inc.jl

```diff
@@ -488,6 +724,91 @@ function gamma_inc_fsum(a::Float64, x::Float64)
     return (1.0 - q, q)
 
 end
+"""
+    gamma_inc_inv_psmall(a,p)
+
+Compute x0 - initial approximation when `p` is small.
+"""
+function gamma_inc_inv_psmall(a::Float64, p::Float64)
+    logr = (1.0/a)*(log(p) + logabsgamma(a + 1.0)[1])
+    r = exp(logr)
+    m=0
+    a2=a*a
+    a3=a2*a
+    a4=a3*a
+    ap1=a + 1.0
+    ap12=(a+1.0)*ap1
+    ap13=(a+1.0)*ap12
+    ap14=ap12*ap12
```

Comment:
and similarly here
```suggestion
    ap1⁴ = ap1²*ap1²
```
etc.